wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -764,16 +764,16 @@ def icon(name=None, classname=None, title=None, wrapped=False, class_name=None):
 
         warn(
             (
-                "Icon template tag `class_name` has been renamed to `classname`, please adopt the new usage instead.",
-                f'Replace `{{% icon ... class_name="{class_name}" %}}` with `{{% icon ... classname="{class_name}" %}}`',
+                "Icon template tag `class_name` has been renamed to `classname`, please adopt the new usage instead. "
+                f'Replace `{{% icon ... class_name="{class_name}" %}}` with `{{% icon ... classname="{class_name}" %}}`'
             ),
             category=RemovedInWagtail50Warning,
         )
 
     return {
         "name": name,
         # supporting class_name for backwards compatibility
-        "classname": classname or class_name or "icon",
+        "classname": classname or class_name or "",
```

Comment:
Isn't this a potentially breaking change? The classname has defaulted to `"icon"` since at least 3.0: https://github.com/wagtail/wagtail/blob/stable/3.0.x/wagtail/admin/templatetags/wagtailadmin_tags.py#L681

Reply:
Answered in https://github.com/wagtail/wagtail/pull/9920#discussion_r1081655305 :-)

Reply:
But the template already has an `icon` class in it?

https://github.com/wagtail/wagtail/blob/357edf2914bfb7b1265011b1009f0d57ff9824f1/wagtail/admin/templates/wagtailadmin/shared/icon.html#L4

Adding `or "icon"` would add an extraneous `icon` class to the rendered HTML. I can't see why this would be a breaking change but I'm happy to revert this (or maybe remove the hardcoded class in the HTML, but I think *that* would potentially be breaking).

Reply:
I take it back... looks like there are tests that *rely on* this fact (they expect the HTML to contain the double `icon` classes). I'll just revert this and revisit separately if necessary.